javascript/javascript1/week1/housey pricey/housey-pricey.js

```diff
@@ -0,0 +1,21 @@
+
+
+name = "Peter";
+const volumeInMetersPeter = (8 * 10 * 10);
+const gardenSizeInM2Peter = 100;
+housePrice = (volumeInMetersPeter * 2.5 * 1000 + gardenSizeInM2Peter * 300);
+
+console.log("Peter paid 2.500.000 while based on the above calculation the house price is " + housePrice + ". So, he paid too much. :\ ")
+
```

Comment:
To calculate whether this has been a good buy or not then use Javascript logic to determine whether it was a good buy or a bad buy.

Hint: `if (housePrice is less than 2500000) then...`

```
if (good buy condition) {
   console.log('Good buy message here');
} else {
   console.log('Bad buy message here');
}
```